roles/opendevshop.devmaster/tasks/config-devmaster.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- name: Devmaster | Save GitHub Token
```

Comment:
For some reason, I like the lowercase version better. Let's keep it "Devmaster" so it remains consistent with how it is formatted everywhere else.